client/app/pages/dashboards/DashboardPage.jsx

```diff
@@ -184,8 +185,15 @@ function DashboardControl({ dashboardOptions }) {
   const showFullscreenButton = !dashboard.is_draft;
   const showShareButton = dashboard.publicAccessEnabled || (canEditDashboard && !dashboard.is_draft);
   const showMoreOptionsButton = canEditDashboard;
+
+  const unarchiveDashboard = () => {
+    recordEvent("unarchive", "dashboard", dashboard.id);
+    updateDashboard({ is_archived: false }, false);
```

Comment:
@gabrieldutra I've noticed for some of the update operations we don't send dashboard version information. When I did try sending version here, it had out of date version. It seems like we're hiding some flawed logic by avoiding sending the version with all updates.  For example here: after unarchiving a dashboard you can no longer edit it. 

We should address this in a separate PR, but wanted to point out to start the discussion in case I overlooked something.

Reply:
The archive function is the only one that doesn't send dashboard version information, no? Anyway, I think I know what's going on, the update function is set to only update the changed keys from the `dashboard` object. Since the archive method doesn't send the version information, it keeps the outdated version.  

In case you want to make it work for this PR (or lmk if you want me to push a commit), it should be a matter of changing this line to `pick(updatedDashboard, [...keys(data), "version"]`:
https://github.com/getredash/redash/blob/9eaa44da4a998edff767dccd18d59e5796913200/client/app/pages/dashboards/hooks/useDashboard.js#L72